docs/widgets.md

```diff
@@ -159,6 +159,35 @@ Example := Window {
 }
 ```
 
+## `TextEdit`
+
```

Comment:
This could do with a description that outlines the current basic feature set (and maybe known limitations).

Reply:
i added some text there